src/app/components/MostRead/Canonical/Item/index.tsx

```diff
@@ -46,14 +46,14 @@ export const MostReadLink = ({
   size,
   eventTrackingData,
 }: PropsWithChildren<MostReadLinkProps>) => {
-  const clickTrackerHandler = useClickTrackerHandler(eventTrackingData);
+  const clickTrackerHandlerProp = useATIClickTrackerHandler(eventTrackingData);
```

Comment:
Let's remove `prop` from the name

```suggestion
  const clickTrackerHandler = useATIClickTrackerHandler(eventTrackingData);
```

Reply:
We could even consider including ATI in the name, to make it reallly explicit (and an example to follow for other components which we want to add click tracking to?)

```suggestion
  const atiClickTrackerHandler = useATIClickTrackerHandler(eventTrackingData);
```